modules/ui/photoviewer.js

```diff
@@ -75,30 +75,34 @@ export function uiPhotoviewer(context) {
 
 
         function setPhotoFromViewerButton() {
-            if (services.mapillary.isViewerOpen()) {
-                if (context.mode().id !== 'select' || !(layerStatus('mapillary') && getServiceId() === 'mapillary')) {
+            const service = getServiceId();
+            const isActiveService = ['mapillary', 'panoramax'].includes(service) && services[service].isViewerOpen();
```

Comment:
Lets extract this array into something like a `const PHOTO_SERVICES_WITH_TAG_BUTTON = []` to give it a bit more visibility that this is the place where we configure the services. Please check the code conventions on such const (naming in placement in file); I don't know how that is done elsewhere in the codebase.

We are also making the assumption here that the service name and tag key will be the same. That is true for those two but might be different for others. We could account for this now… but it feels like premature optimization so I would do it in a refactoring later when the need comes up.